core/src/main/java/io/undertow/UndertowMessages.java

```diff
@@ -610,4 +610,8 @@
 
     @Message(id = 196, value = "Session with id %s already exists")
     IllegalStateException sessionWithIdAlreadyExists(String sessionID);
+
+    @Message(id = 199, value = "Invalid ACL entry")
+    IllegalArgumentException invalidACLAddress(@Cause Exception e);
+
```

Comment:
As I mentioned at https://github.com/undertow-io/undertow/pull/1160#discussion_r710035772, I think this part causes merge conflict.